content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md

```diff
@@ -11,104 +11,241 @@ keywords:
 - data backup and recovery
 ---
 
-Google Photos works best in TrueNAS using a Google Photos API key and [rclone](https://rclone.org/) token.
+Google Photos cloud sync tasks in TrueNAS SCALE use the [rclone](https://rclone.org/) backend for the [Google Photos API](https://developers.google.com/photos) to authenticate credentials and transfer data.
+
+Configuring a Google Photos cloud sync task is a multi-part procedure that includes:
+
+1. [Planning your deployment](#before-you-begin) and selecting a local dataset.
+2. [Generating Google API credentials](#creating-the-api-credentials) on the Google Cloud API dashboard.
+3. [Installing rclone and generating a token](#configuring-rclone) on your remote client OS.
+4. [Adding Google Photos cloud credentials](#adding-google-photos-cloud-credentials) on TrueNAS SCALE.
+5. [Configuring the cloud sync task](#creating-the-cloud-sync-task) on SCALE.
```

Comment:
This is still reading like a bullet list and not like a summary procedure so I suggest this change:
```suggestion
Configuring a Google Photos cloud sync task is a multi-part procedure where you:

1. [Plan your deployment](#before-you-begin) and selecting a local dataset.
2. [Generate Google API credentials](#creating-the-api-credentials) on the Google Cloud API dashboard.
3. [Install rclone and generate a token](#configuring-rclone) on your remote client OS.
4. [Add Google Photos cloud credentials](#adding-google-photos-cloud-credentials) on TrueNAS SCALE.
5. [Configure the cloud sync task](#creating-the-cloud-sync-task) on SCALE.
```